admin_manual/occ_command.rst

```diff
@@ -1362,16 +1362,21 @@ The ``--dry-run`` option can be used to simulate the restore without actually re
 User commands
 -------------
 
-The ``user`` commands create and remove users, reset passwords, display a simple
+The ``user`` commands create and remove users, reset passwords, manage authentication tokens / sessions, display a simple
 report showing how many users you have, and when a user was last logged in::
 
  user
   user:add                            adds a user
-  user:add-app-password               adds a app password named "cli"
+  user:add-app-password               adds a app password named "cli" (deprecated: alias for user:auth-tokens:add)
+  user:auth-tokens:add                Add app password for the named account
+  user:auth-tokens:delete             Deletes an authentication token
+  user:auth-tokens:list               List authentication tokens of an user
```

Comment:
Just to be rephrased similarly to options below
```suggestion
  user:auth-tokens:list               List specified user authentication tokens

Reply:
```
occ user:auth-tokens:list   --help
Description:
  List authentication tokens of an user

```

If this should be reworded, it should be reworded in server first and then simply be aligned here

Reply:
Yeah more polish would certainly be nice on the embedded command descriptions. I'd like to go through most of them one day, but that's a bigger job. I don't have the time or patience for it right now, particularly relative to more pressing priorities.